TYPO3.Neos/Classes/TYPO3/Neos/ViewHelpers/Backend/UserInitialsViewHelper.php

```diff
@@ -74,9 +75,9 @@ public function render($format = 'initials')
 
         switch ($format) {
             case 'initials':
-                return mb_substr($requestedUser->getName()->getFirstName(), 0, 1) . mb_substr($requestedUser->getName()->getLastName(), 0, 1);
+                return mb_substr(preg_replace('/[^[:alnum:][:space:]]/u', '', $requestedUser->getName()->getFirstName()), 0, 1) . mb_substr($requestedUser->getName()->getLastName(), 0, 1);
```

Comment:
I didn't realize this the first time around, but now I wonder: Why do you remove special characters from the first name but not from the last name?

Reply:
good point, should be done to both.. fixed